Codeinwp/neve
# fix: translations not loading properly for scripts [ref #2290]

### Summary
This aims to fix the translation issue with the scripts found in the theme. 


### Will affect visual aspect of the product

NO

### Test instructions


- Pick a language that has 100% translations on WP.org (https://translate.wordpress.org/projects/wp-themes/neve/)
- Set your website language to that locale 
- Customizer and dashboard strings should be translated properly now
- Loco translate should work fine on those 

*Known issues:*
- Sidebar script does not translate (no idea why at this point);
-  Top / Right / Bottom / Left were not translatable for the sizing input - fixed that here (but they don't have translations assigned);
- Search placeholder for header presets was not properly wrapped in `__` function;


Ref #2290.
